testing/upload_validation.ts

```diff
@@ -1114,4 +1115,36 @@ const legacyPackMetadataSchema = validateFormulas(
         "Specify the domain that your pack makes http requests to using `networkDomains: ['example.com']`",
       path: ['networkDomains'],
     },
-  );
+  )
+  .superRefine((data, context) => {
```

Comment:
NIt, is `data` typed as `any` here? I think it's safe to cast it at this point in the validation flow, doing so may help catch future refactoring issues.

Reply:
Added a cast